File: helloPerson.cpp
#include <iostream>
#include <string>

class person{
    std::string title, firstName, lastName;
    int age;

    public:
        person(std::string title, std::string firstName, std::string lastName, int age) {
            this->title = title;
            this->firstName = firstName;
            this->lastName = lastName;
            this->age = age;
        }

        void greetInformal() {
            std::cout << "Hello, " << this->title << " " << this->lastName << "!\n";
        }

        void greetFormal() {
            std::cout << "Hello, " <<this->title << " " <<this->lastName << "!\n";
        }

        void howOld() {
            std::cout <<this->firstName << " is " <<this->age << " years old.\n";
        }

        void birthday() {
            this->age += 1;
            std::cout << "Happy Birthday, " << this->firstName << "! You are now " << this->age << " years old.\n";
        }
};

int main() {
    std::string title, firstName, lastName;
    int age;

    std::cout << "What is your title? ";
    std::cin >> title;
    std::cout << "What is your first name? ";
    std::cin >> firstName;
    std::cout << "What is your last name? ";
    std::cin >> lastName;
    std::cout << "How old are you? ";
    std::cin >> age;

    person you(title, firstName, lastName, age);

    you.greetInformal();
    you.greetFormal();
    you.howOld();
    you.birthday();
    you.howOld();

    return 0;
}

File: helloPerson.py
class Person:
    def __init__(self, title = "Private First Class ", firstName = "Forrest", lastName = "Gump", age = 23):
        self.title = title
        self.firstName = firstName
        self.lastName = lastName
        self.age = age
        
    def greetInformal(self):
        print("Hello, ", self.firstName, "\b!")
        
    def greetFormal(self):
        print("Hello,", self.title, self.lastName, "\b!")
    
    def howOld(self):
        print(self.firstName, "is ", self.age, " years old!")
        
    def birthday(self):
        self.age += 1
        print("Happy Birthday, ", self.firstName, "\b! You are now ", self.age, " years old!")
        
title = input("What is your title? ")
firstName = input("What is your first name? ")
lastName = input("What is your last name? ")
age = input("How old are you? ")

you = Person(title, firstName, lastName, int(age))

you.greetInformal()
you.greetFormal()
you.howOld()
you.birthday()
you.howOld()

you.age -= 1
you.howOld()
